src/main/java/io/cdap/plugin/gcp/dataplex/sink/DataplexBatchSink.java

```diff
@@ -357,18 +357,17 @@ protected void initOutput(BatchSinkContext context, BigQuery bigQuery, String ou
     Configuration configuration = new Configuration(baseConfiguration);
 
     // Build GCS storage path for this bucket output.
+    DatasetId datasetId = DatasetId.of(datasetProject, dataset);
     String temporaryGcsPath = BigQuerySinkUtils.getTemporaryGcsPath(bucket, runUUID.toString(), tableName);
-    BigQuerySinkUtils.configureOutput(configuration,
-      DatasetId.of(datasetProject, dataset),
-      tableName,
-      temporaryGcsPath,
-      fields);
+    BigQuerySinkUtils.configureOutput(configuration, datasetId, tableName, temporaryGcsPath, fields);
     // Both emitLineage and setOutputFormat internally try to create an external dataset if it does not already exist.
     // We call emitLineage before since it creates the dataset with schema which is used.
     List<String> fieldNames = fields.stream()
       .map(BigQueryTableFieldSchema::getName)
       .collect(Collectors.toList());
-    BigQuerySinkUtils.recordLineage(context, outputName, tableSchema, fieldNames);
+    String fqn = BigQueryUtil.getFqn(datasetProject, dataset, config.getTable());
+    String location = bigQuery.getDataset(datasetId).getLocation();
+    BigQuerySinkUtils.recordLineage(context, new io.cdap.plugin.common.Asset(fqn, location), tableSchema, fieldNames);
```

Comment:
import the package for Asset

Reply:
Asset is already imported from dataplex api:
https://github.com/data-integrations/google-cloud/blob/7abc79d048d2e6ff22a7aa6b52d81188d4c5d823/src/main/java/io/cdap/plugin/gcp/dataplex/sink/DataplexBatchSink.java#L29